tools/install_bazelisk.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+
+if ! command -v sudo >/dev/null; then
+    SUDO=
+else
+    SUDO=sudo
+fi
+
+# renovate: datasource=datasource depName=bazelbuild/bazelisk
+BAZELISK_VERSION=v1.20.0
+
+echo "Checking if Bazelisk ${BAZELISK_VERSION} needs to be installed..."
+if [[ $(command -v bazel) && "$(bazel version | grep 'Bazelisk' | cut -d ' ' -f 3)" = "${BAZELISK_VERSION}" ]]; then
+    echo "Bazelisk ${BAZELISK_VERSION} already installed, skipping."
+else
+    BAZEL=$(command -v bazel)
+    if [ -n "${BAZEL}" ] ; then
+        echo "Removing old Bazel version at ${BAZEL}"
+        ${SUDO} rm "${BAZEL}"
+    else
+        BAZEL=/usr/local/bin/bazel
+    fi
+    OS=$(uname -s | tr '[:upper:]' '[:lower:]')
+    ARCH=$(uname -m) && [ "$ARCH" != "x86_64" ] || ARCH="amd64"
```

Comment:
This will not work for `uname -m` returning `aarch64`, which will need to be translated to `arm64`, like we did for `install_bazel.sh`. Seems `bazelisk` needs `x86_64` to be translated to `amd64` (like you did here) while `bazel` did not.

Reply:
Updated and thanks.